contracts/src/steel/Steel.sol

```diff
@@ -84,12 +83,21 @@ library Beacon {
     /// @dev https://eips.ethereum.org/EIPS/eip-4788
     address internal constant BEACON_ROOTS_ADDRESS = 0x000F3df6D732807Ef1319fB7B8bB8522d0Beac02;
 
+    /// @notice call to the EIP-4788 beacon roots contract failed due to an invalid block timestamp.
+    /// @dev A block timestamp is invalid if it does not correspond to a stored block on the
+    ///      EIP-4788 contract. This can happen if the timestamp is too old, and the block
+    ///      it corresponds too has been evicted from the cache, if the timestamp corresponds to a
+    ///      slot with no block, or if the timestamp does not correspond to a slot at all.
```

Comment:
```suggestion
    /// @notice Call to the EIP-4788 beacon roots contract failed due to an invalid block timestamp.
    /// @dev A block timestamp is invalid if it does not correspond to a stored block on the
    ///      EIP-4788 contract. This can happen if the timestamp is too old, and the corresponding 
    ///      block has been evicted from the cache, if the timestamp corresponds to a
    ///      slot with no block, or if the timestamp does not correspond to any slot at all.
```